services/suspension-service/tasks

```diff
@@ -0,0 +1,310 @@
+# Terraform Functions #
+#######################
+
+function tf_init {
+  check_env
+  cd terraform
+
+  terraform init -reconfigure \
+    -backend-config key="${NHS_SERVICE}-${NHS_ENVIRONMENT}/terraform.tfstate" \
+    -backend-config bucket="prm-deductions-${NHS_ENVIRONMENT}-terraform-state" \
+    -backend-config dynamodb_table="prm-deductions-${NHS_ENVIRONMENT}-terraform-table" \
+    -backend-config region=${AWS_DEFAULT_REGION}
+}
+
+function tf_plan {
+  operation=$1
+
+  if [[ -z "${IMAGE_TAG}" ]]; then
+    set_image_tag
+  fi
+
+  tf_init
+
+  terraform get # modules
+  if [[ "${operation}" == "create" ]]; then
+    terraform plan -var task_image_tag=$IMAGE_TAG -var-file="configs/${NHS_SERVICE}/${NHS_SERVICE}.tfvars" -var-file="configs/${NHS_SERVICE}/$NHS_ENVIRONMENT.tfvars" -out="${NHS_SERVICE}.tfplan"
+  elif [[ "${operation}" == "destroy" ]]; then
+    terraform plan -var task_image_tag=$IMAGE_TAG -var-file="configs/${NHS_SERVICE}/${NHS_SERVICE}.tfvars" -var-file="configs/${NHS_SERVICE}/$NHS_ENVIRONMENT.tfvars" -out="${NHS_SERVICE}.tfplan" -destroy
+  else
+    echo "Unknown operation (should be create or destroy), got: ${operation}"
+    exit 1
+  fi
+}
+
+function tf_apply {
+  tf_init
+  terraform get # modules
+  terraform apply ${NHS_SERVICE}.tfplan
+  terraform output -json > tf-out.json
+}
+
+function configure_local_envs {
+  export LOCALSTACK_URL="http://localhost:4566"
+  export AWS_REGION="eu-west-2"
+  export AWS_DEFAULT_REGION="eu-west-2"
+  export S3_BUCKET_NAME="test-bucket"
+  export DYNAMODB_NAME="local-test-db"
+  export DYNAMODB_LOCAL_ENDPOINT="http://localhost:4566"
+  export AWS_ACCESS_KEY_ID="LSIA5678901234567890"
+  export AWS_SECRET_ACCESS_KEY="LSIA5678901234567890"
+  export NHS_ENVIRONMENT="local"
+}
+
+function run_localstack_local {
+  docker compose -f docker-compose.localstack-local.yaml up -d
+}
+
+function destroy_localstack {
+  echo docker processes running: $(docker ps)
+
+  DOCKER_CONTAINERS_ON_PORT=$(docker ps --filter publish=4566 -q)
+
+  echo docker processes running on expected localstack port: $DOCKER_CONTAINERS_ON_PORT
+
+  if [ -z "$DOCKER_CONTAINERS_ON_PORT" ]; then
+    echo no localstack nonsense going on, cool
+  else
+    echo trying to kill localstack...
+    docker stop $DOCKER_CONTAINERS_ON_PORT
+  fi
+}
+
+###########
+## TASKS ##
+###########
+
+command="$1"
+case "${command}" in
+  fetch_utils)
+      fetch_redaction_utils
+      ;;
+  destroy_localstack)
+      destroy_localstack
+      ;;
+  build_docker)
+      configure_envs
+      ./tasks _build
+      configure_docker_repository_uri
+      fetch_redaction_utils
+      build_docker_image
+      echo "Pushing the Docker image... $REPOSITORY_URI:latest $REPOSITORY_URI:$IMAGE_TAG"
+      docker push $REPOSITORY_URI:$IMAGE_TAG
+      ;;
+  _build)
+      rm -rf build/
+      ./gradlew assemble
+      ;;
+  build)
+      ./tasks _build
+      ;;
+  _test_unit)
+      ./gradlew test
+      ;;
+  test_unit)
```

Comment:
We can clean up this tasks file massively. The non-underscored functions (such as `test_unit` here) I believe were written originally so that they shell into Dojo and then call the underscored functions (in this case `_test_unit`). Now we've moved away from Dojo, we can get rid of the duplicated functions.